### ISSUE TITLE
Accessibility: MAS17: Color Contrast: [All]: Text is not meeting minimum Luminosity contrast ratio of 4.5:1 against background.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#28]
#A11yMAS
#MAS17
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
**Scenario-1:**
1. Open https://code.visualstudio.com/ in EDGE.
2. Check color contrast ratio for the "IntelliSense", "Debugging", "Built-in Git" and "Extensions" links available below the "Code Editing. Redifined" image.
3. Observe the ratio in color contrast analyser

**Scenario-2:**
1. Open  https://code.visualstudio.com/ in EDGE.
2. Check color contrast ratio for the "Follow @ Code" Twitter link in the footer of the page.
3. Observe the ratio in color contrast analyser

**Same issue is observed with the links**
https://code.visualstudio.com/docs
https://code.visualstudio.com/docs/introvideos/overview
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/languages/overview
https://code.visualstudio.com/docs/setup/setup-overview
https://code.visualstudio.com/docs/runtimes/nodejs

**Actual Result:**
Luminosity contrast ratio is less than 4:5:1 for the text against the background.

**Expected Result:**
Luminosity contrast ratio should be above 4:5:1 for all the text against the background.

**User Impact:**
People with low vision often have difficulty reading text or buttons that do not contrast with their background. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a minimum luminance contrast ratio between the text, including placeholder text, and its background can make the text more readable even if the person does not see the full range of colors. It also works for individuals who see no color. 

**Screenshots:**
![color contrast scenario 1](https://cloud.githubusercontent.com/assets/24825441/21709529/a6ecd578-d407-11e6-8a03-21c4510fa16d.png)
![color contrast scenario 2](https://cloud.githubusercontent.com/assets/24825441/21709533/aa2a5490-d407-11e6-9f8e-6be0b19c86f6.png)
![color contrast scenario 3](https://cloud.githubusercontent.com/assets/24825441/21709536/ac7aee26-d407-11e6-8c8f-2da65cd724a8.png)

**Comment by waderyan** [comment]
Fixed for IntelliSense, Debugging, etc. icons 

![image](https://cloud.githubusercontent.com/assets/2449568/26737495/38a6baca-477f-11e7-9f87-951f6864e6a1.png)

**Comment by waderyan** [comment]
Fixed for Windows logo

![image](https://cloud.githubusercontent.com/assets/2449568/26737468/19e7e05a-477f-11e7-8db4-ba37bb3bef46.png)

**Comment by chrisdias** [comment]
fix for follow button: https://github.com/Microsoft/vscode-website/commit/378fee8dfc0d791373f90aa2ef797ac996a0e3d2
